deltachat-ios/AppDelegate.swift

```diff
@@ -173,6 +174,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         // }
 
         switch url.scheme?.lowercased() {
+        case "dcaccount":
+            _ = self.dcAccounts.add()
```

Comment:
Why don't you implement it similar to openpgp4fpr or handleMailtoURL, delegating it to appCoordinator?

Reply:
just tried to make it work at all for now :) , if I were to make it nice I would show the confirmation dialog first regardless of screen and don't even create the account if the user aborts/cancels. ;)